tendermint-proto/tests/unit.rs

```diff
@@ -0,0 +1,10 @@
+#[test]
+pub fn import_evidence_info() {
```

Comment:
What are we asserting with this test?

Reply:
I just didn't want to submit the generated files with no test at all. It's a weak try at "at least do something" with the generated structs.

We can either implement decvent tests for all structs or completely remove testing the automated structs. I'm not sure what's the best way of dealing with it, so this is a reminder to discuss it. :)

Reply:
I'd ditch this test and capture a follow-up issue or inform an existing which already tracks testing. The challenge here is to find the right level of granularity. As this mostly around serialisation it might make more sense to test it indirectly through other means.